src/apps/ENGINE/s_dbg_sourceview.cpp

```diff
@@ -532,8 +532,8 @@ bool SOURCE_VIEW::OpenSourceFile(const char *_filename)
         CDebug.SaveRecentFileALine(SourceFileName, nActiveLine);
     }
 
-    char DirectoryName[MAX_PATH];
-    fio->_GetCurrentDirectory(sizeof(DirectoryName), DirectoryName);
+    char DirectoryName[MAX_PATH * 4];
```

Comment:
it doesn't seem like a proper solution. I'd propose to replace char[] with std::string

Reply:
Check new version please.